pkgs/servers/home-assistant/custom-components/emporia_vue/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, buildHomeAssistantComponent
+, pyemvue
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "presto8";
+  domain = "emporia_vue";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "magico13";
+    repo = "ha-emporia-vue";
+    rev = "master";
```

We cannot use either branch directly since then the hash would change as soon as a new commit would be done.

```suggestion
    rev = "v${version}";
```